### DISCUSSION TITLE
Rook external to Proxmox Ceph

### DISCUSSION THREAD

**Discussion by urbaman** [rook/rook#13680] [Q&A]
Hi,

Trying to get Ceph external go to use Proxmox Ceph as K8s Storage.

Followed the guide, but still some namespace misconfiguration is happening.

0) The operator and all rbd/cephfs pods are running in rook-ceph namespace
1) The Cluster is HEALTH_OK in rook-ceph-external namespace
2) The Toolbox, from the same namespace as the Cluster (rook-ceph-external) has permission problems running ceph status
3) PVCs have problems because che csi plugins search for resources in the wrong namespace:

`rook-ceph.rbd.csi.ceph.com_csi-rbdplugin-provisioner-5d46874df5-7t54b_9bffc94a-7596-4fea-94e0-965a5b3b9211  failed to provision volume with StorageClass "ceph-rbd": error getting secret rook-csi-rbd-provisioner in namespace external-rook: secrets "rook-csi-rbd-provisioner" not found`

I do not have any external-rook namespace.

**Comment by urbaman** [comment]
Ok, it was the auto generated storage classes: they pointed to the wrong namespace. Created new storage classes and everything went up.

Still, the toolkit do not work as expected.

**Reply by travisn** [reply]
@urbaman #13681 has the details to resolve this, right?

**Reply by urbaman** [reply]
Yes! At least the storage class, not the toolbox.
